packages/runner-ct/src/specs/specs-list.tsx

```diff
@@ -24,22 +24,15 @@ export const SpecsList: React.FC<SpecsListProps> = observer(
         </header>
         <div className="specs-list-search-input-container">
           <input
-            placeholder='Select tests to run...'
+            placeholder='Find a spec'
             value={state.specSearchText}
             onChange={(e) => state.setSearchSpecText(e.currentTarget.value.toLowerCase())}
           />
         </div>
         <div className="specs-list-scroll-container">
           <ul className="specs-list-container">{
-            specGroups.map((item) => {
-              { // The `active` prop here is used only to
```

Comment:
there is no active prop, seems this is an old comment

Reply:
Yes !! please remove it ;)